flink-ai-flow/ai_flow/api/workflow_operation.py

```diff
@@ -52,16 +55,23 @@ def _upload_project_package(workflow: Workflow):
     workflow.properties[WorkflowPropertyKeys.BLOB] = current_project_config().get(WorkflowPropertyKeys.BLOB)
 
 
-def _set_entry_module_path(workflow: Workflow, entry_module_path: Text):
+def _set_entry_module_path(workflow: Workflow, entry_module_path: Jsonable):
```

Comment:
Why change the type of `entry_module_path` to `Jsonable`? 
I find the `_apply_full_info_to_workflow()` still calls `_set_entry_module_path` with a Text `entry_module_path`.